### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by omkar200412**
some changes of ui/ux and color

### CODE REVIEW

**Review by MarshallOfSound — CHANGES_REQUESTED** [review]
What is the purpose of this change, is their a bug somewhere or a UX / A11Y issue you're trying to fix?

**Review by omkar200412 — COMMENTED** [review]
some changes in ui/ux nad colors

**Comment by omkar200412** [comment]
some changes of ui/ux color

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "lint": "prettier --check \"src/**/*.js\" \"src/**/*.css\"",
     "prettier:write": "prettier --write \"src/**/*.js\" \"src/**/*.css\"",
     "postinstall": "cp node_modules/prismjs/themes/*.css src/static/css/prismjs",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "dev": "nodemon index.js"
```
Comment (omkar200412) [inline package.json:47]: Is there are some npm issue or I'm not able to open this project in web so I'll try Nodemon then issue solved

**Comment by codebytere** [comment]
Closing as invalid.

**Comment by omkar200412** [comment]
ok No problem I'll come Next time  with New bug

**Pull request closed without merging**